interface/main/tabs/js/tabs_view_model.js

```diff
@@ -131,7 +136,7 @@ function navigateTab(url,name,afterLoadFunction)
     }
     else
     {
-        curTab=new tabStatus(xl('New'),url,name,true,false,false);
+        curTab=new tabStatus(xl("Loading") + " <a style='font-size:80%'><i class='fa fa-spinner fa-pulse'></i></a>",url,name,label,true,false,false);
```

Comment:
we really don't need to wrap icon in an anchor as it doesn't persist or shouldn't.
also, get rid of style 80% and add fa-sm class to icon. Remember that fa size will inherit the parent font-size, ie whatever tab div wrappers font-size so fa-1x will be whatever div font-size is say 1em resulting in 1 X 1 and fa-sm would be 1em x .875 etc.. I'm unsure if BS4 has the fa-sm class but it still will inherit default fa-1x of parent if not etc.
 
`curTab=new tabStatus(xl("Loading") + " <i class='fa fa-spinner fa-pulse fa-sm'></i>",url,name,label,true,false,false);`

Reply:
Cant find `fa-sm` in the font awesome 4.7.0

Reply:
I think i found a fix and room for impovement😄.